tests/log.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"io"
+	"os"
+
+	"go.uber.org/zap/zapcore"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+)
+
+// Define a simple encoder config appropriate for testing
+var simpleEncoderConfig = zapcore.EncoderConfig{
+	TimeKey:       "",
+	LevelKey:      "level",
+	NameKey:       "",
+	CallerKey:     "",
+	MessageKey:    "msg",
+	StacktraceKey: "stacktrace",
+	// TODO(marun) Figure out why this is outputting e.g. `Level(-6)` instead of `INFO`
+	EncodeLevel: zapcore.LowercaseLevelEncoder,
```

Comment:
We have modified how levels are encoded. We should be providing a custom leveler like in the rest of avalanchego:
```golang
func levelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
	enc.AppendString(Level(l).String())
}

func jsonLevelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
	enc.AppendString(Level(l).LowerString())
}
```
See `utils/logging/format.go#newTermEncoderConfig`

Reply:
Nope - makes sense to me.